Marlin/Configuration.h

```diff
@@ -433,6 +433,64 @@ const bool Z_MIN_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the l
 //#define Z_PROBE_SLED
 //#define SLED_DOCKING_OFFSET 5 // The extra distance the X axis must travel to pickup the sled. 0 should be fine but you can push it further if you'd like.
 
+// With this option servos are powered only during movement, then turned off to prevent jitter.
+//#define DEACTIVATE_SERVOS_AFTER_MOVE
+
+
+// With probe what is triggered when stowed we can test if it is deployed/stowed or not.
+// This is common for Allen Key probes and some servo probes.
+//#define PROBE_IS_TRIGGERED_WHEN_STOWED_TEST
+
+// Allen key retractable z-probe as seen on many Kossel delta printers - http://reprap.org/wiki/Kossel#Automatic_bed_leveling_probe
```

Comment:
Added Allen key probe to main Configuration.h .  Now we can test Allen key probes with Cartesian printers.